python/cudf/cudf/core/reshape.py

```diff
@@ -394,11 +402,24 @@ def get_dummies(
         columns. Note this is different from pandas default behavior, which
         encodes all columns with dtype object or categorical
     dtype : str, optional
-        output dtype, default 'int8'
-    """
-    if dummy_na:
-        raise NotImplementedError("dummy_na is not supported yet")
+        output dtype, default 'uint8'
 
+    Exmaples
+    --------
+    >>> import cudf
+    >>> df = cudf.DataFrame({"a": ["value1", "value2", None], "b": [0, 0, 0]})
+    >>> cudf.get_dummies(df)
+       b  a_value1  a_value2
+    0  0         1         0
+    1  0         0         1
+    2  0         0         0
+
+    >>> cudf.get_dummies(df, dummy_na=True)
+       b  a_None  a_value1  a_value2
```

Comment:
Is `a_None` the naming scheme we want to use? Pandas uses `a_nan` it looks like for example. Given we can have both `NaN` and `null` values, I'd propose we use `a_nan` for `NaN` values and `a_null` for null values.